package.json

```diff
@@ -79,7 +80,9 @@
     "events": "^3.3.0",
     "find-replacement-tx": "^1.2.3",
     "firebase": "^9.8.2",
+    "framer-motion": "^10.12.4",
```

Comment:
we already have `react-spring`. can we remove 1?

Reply:
will make improvements for this later, I will consider removing one of them